packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -33,7 +33,9 @@ function expectEqualStates(a: StreamState, b: StreamState) {
   expect(StreamUtils.serializeState(a)).toEqual(StreamUtils.serializeState(b))
 }
 
-describe('IPFS caching', () => {
+const describeIfV3 = process.env.CERAMIC_ENABLE_V4_MODE ? describe.skip : describe
```

Comment:
here's an example of how to selectively enable/disable a suite of tests based on which mode we're in.  A very similar thing could be done to make a `testIfV3` to selectively disable just a single test instead of a whole suite.